internal/config/moduleconfig/module_config.go

```diff
@@ -107,6 +138,10 @@ func LoadModuleConfig(filePath string) (ModuleConfig, error) {
 		log.Fatal("")
 	}
 
+	if !ValidateZeroVersion(config) {
+		constraint := config.ZeroVersion.Constraints.String()
+		return config, errors.New(fmt.Sprintf("Module's zero requirement not satisfied: expected %s  received: %s", constraint, version.AppVersion))
```

Comment:
Let's add some more details, we are telling them a bit about what's wrong but not how to fix it, and it will be the user who is faced with this message.

Reply:
how about this 
![image](https://user-images.githubusercontent.com/1787590/112392140-53fadb00-8ccf-11eb-9104-92b5b2d405bd.png)